lib/test/tester.js

```diff
@@ -117,7 +117,10 @@ class Tester {
       return
     }
 
-    const editorRanges = _.filter(this.ranges, (r) => { return _.endsWith(file, r.file.replace(/^_\//g, '')) })
+    const re = /[^/\\]+$/g
+    const editorRanges = _.filter(this.ranges, (r) => {
+      return file.match(re)[0] === r.file.match(re)[0] && !_.endsWith(file, '_test.go')
```

Comment:
Why is the check for _test.go necessary? It was not previously...

If it's really necessary then move it out of the `filter`.